.github/workflows/eerelease.yml

```diff
@@ -91,29 +92,20 @@ jobs:
       - name: Create the latest tag
         if: ${{ inputs.tag_as_latest }}
         run: |
-          podman tag "${IMAGE_ID}" ghcr.io/ansible-community/${{ inputs.ee-name }}:latest
+          podman tag "${IMAGE_ID}" ghcr.io/${{ env.IMAGE_NAME }}:latest
           podman images
 
-      - name: Login to the Github Container registry
-        uses: redhat-actions/podman-login@v1
-        with:
-          registry: ghcr.io/${{ github.repository_owner }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Upload artifact EE to ghcr
         uses: redhat-actions/push-to-registry@v2
         with:
-          image: ${{ inputs.ee-name }}
-          tags: ${{ env.EE_VERSION }}
-          registry: ghcr.io/ansible-community
+          image: ${{ env.IMAGE_NAME }}
+          tags: >-
```

Comment:
Can you please me how this works? There are occasions where we release images which are different tag than `latest`. Will the change cover this one as well?

Reply:
`>-` makes the entire thing into one line on the YAML parser level, the lines are concatenated with a whitespace. The first part is `${{ env.EE_VERSION }}` which is injected unconditionally. The second one is an expression `${{ inputs.tag_as_latest && 'latest' || '' }}`. When evaluated, it'll either be `latest` or an empty string. So the entire `tags` input will contain the EE version, optionally followed by `latest`.
The `redhat-actions/push-to-registry` action accepts a whitespace-separated list of tags. If there's more than one, they'll be applied in one go. This eliminates the need to call the action twice since the dynamic part is the tag that's going to be used.